assets/css/admin.scss

```diff
@@ -219,8 +231,22 @@
 			margin-bottom: 0;
 		}
 
+		.wcs-logos-container {
```

Comment:
Looks good! What do you think about centering the logos when on mobile? 

Before:
![Screen Shot 2021-06-15 at 1 56 48 PM](https://user-images.githubusercontent.com/6723003/122048531-a84ebc00-cde1-11eb-872f-1dd72d1eefa1.png)

After:
![Screen Shot 2021-06-15 at 1 56 54 PM](https://user-images.githubusercontent.com/6723003/122048538-aab11600-cde1-11eb-914e-6ab4d0044c6c.png)

That was done with `justify-content: center`.

Reply:
Looks like a reasonable change, I don't know why it wasn't done before - done in c988f25e1e